src/wallet/wallet_accounts.rs

```diff
@@ -316,7 +316,7 @@ mod tests {
                 &self,
                 _key: SecureStorageKey,
             ) -> Result<Option<Vec<u8>>> {
-                todo!()
+                unreachable!()
```

Comment:
I did not know about `uncreachable` before! it is a more accurate way of crashing than `todo`.

Reply:
This is inside one unit test, a unit test where I only test save, not load, on using that `TestStorage`.